apps/evans/pages/[friend_slug]/[document_slug].tsx

```diff
@@ -42,7 +42,7 @@ export const getStaticProps: GetStaticProps<Props> = async (context) => {
       documentSlug: document.slug,
     })),
   );
-  if (NODE_ENV === `production`) {
```

Comment:
without this, i was spewing .xml files into my `public/` dir whenever i built locally. this preserves building on vercel, the only place we want to write these files.